run_dir/design/flowcell_samples.html

```diff
@@ -39,8 +39,8 @@
   </div>
 </div>
 
-<script src="/static/js/flowcell_samples.js?v={{ gs_globals['git_commit'] }}" id="flowcells-js" data-flowcell="{{ flowcell }}"></script>
 <script src="/static/js/links.js?v={{ gs_globals['git_commit'] }}" id="ln-js" data-workset="{{ flowcell}}"></script>
 <script src="/static/js/running_notes.js?v={{ gs_globals['git_commit'] }}" id="rn-js" data-workset="{{ flowcell }}"></script>
+<script src="/static/js/flowcell_samples.js?v={{ gs_globals['git_commit'] }}" id="flowcells-js" data-flowcell="{{ flowcell }}"></script>
```

Comment:
Awesome! Although this was implemented some time ago, I'm really happy with this solution :+1: